### ISSUE TITLE
Contrubuting

### ISSUE THREAD

**Issue by wellyal** [reactide/reactide#122]
I would like to contribute to this project. Is there a roadmap? Any slack channel?

**Comment by wordyallen** [comment]
ded

**Comment by idchlife** [comment]
Wait, so this a project that is dead and it got like 7k stars? Wow, talk about bubble

**Comment by hubert-associates** [comment]
Well if it is really dead, I'd hope the authors would please tell the 7k star-givers what the scoop is.

**Comment by rajivramv** [comment]
Indeed misleading.
On the flip side, can interested folks get together and make a plan?

On Thu, Aug 17, 2017 at 8:40 PM, Richard G. Hubert <notifications@github.com

**Comment by idchlife** [comment]
I'm wondering why development stopped.
I mean hey, work was done. Even current version has something.

I did not really dig into source code, could it be that it was just probing the market or something? Like hypey title, some design?

**Comment by rajivramv** [comment]
Yep, title + design IMHO too . Trying to get it running went in vain, and
at first look, looks a lot like design than actual code too.

Rajiv Ram V
+919884052643

**Comment by wellyal** [comment]
@idchlife For what I've seen, nothing was really implemented. I ran the code and just a black window was popped

**Comment by idchlife** [comment]
@wellyal about black window, I experienced it too but you can actually see more. There are some issues here that address starting problems (just empty screen), and after that you can choose "new file" or something from the menu and you will see all the interface (which is somewhat working) that was advertised. This slightly working interface is exactly the thing which convinced me at the time this was not a stareater project.

**Comment by basedjux** [comment]
I think what happened is they either

1. Fell apart (The core dev team)
2. Were Star Farming, or
3. Didn't really know what they were doing

I'm more likely to believe 1, or 2, seeing as how they actually created functional code for the project.
Or 1-3 is wrong and they're working on everything locally before they push a load of new stuff.

**Comment by idchlife** [comment]
@basedjux kinda reminds me of https://github.com/spirit/spirit

Still waiting for beta. But spirit at least has maintainer who says he's working hard right now.

Who knows. This could be NoManSky story here. Silence, then boom, whole load of cool stuff.

**Comment by basedjux** [comment]
You know @idchlife that's exactly what I was thinking of... Repeat of NMS haha.
Pure silence then suddenly Foundation Update happened 😛

**Comment by wordyallen** [comment]
I want a dedicated React/React Native IDE more than anyone....I wish they could team up with Nuclide. Despite a dedicated fb team, that ide is still the pits.

**Comment by idchlife** [comment]
lol https://github.com/FermORG/FermionJS

**Comment by basedjux** [comment]
Wait what? Lol. Hang on... Is that the same project? With a different font face?

**Comment by wellyal** [comment]
@idchlife not a big fan of drag and drop solutions.

**Comment by spelunk** [comment]
I'd love to see a continuation of this project.  @bitadj @JiniHendrix @markmarcelo would you consider adding a new maintainer?

**Comment by idchlife** [comment]
I genuinely interested as to what happened to the core team. 2 developers stopped contributing to github somewhat in october and december and last do not respond to our questions about ide.

**Comment by spelunk** [comment]
Safe to assume there won't be further development.  @idchlife if you are interested in building and maintaining a fork, start from taking a look at the open PRs

**Comment by salmon258** [comment]
This is a really great project. I would like to contribute if someone want to continue the dev.

**Comment by simonhochrein** [comment]
We should definitely make a fork and work on that.

**Comment by salmon258** [comment]
@simonhochrein yea, but I think we need someone to lead the project

**Comment by spelunk** [comment]
We can set up an org like reactide-redux or something and give people commit access if interested.  Anyone want to set that up?

**Comment by cristhiandas** [comment]
I'm pretty new with react but I would love to participate, you think that is possible?

**Comment by simonhochrein** [comment]
Should the org name be reactide-project?

**Comment by salmon258** [comment]
To be honest, I have no experience working with open source project. But I'm really excited to contribute on this one, because I really love react!

**Comment by spelunk** [comment]
@simonhochrein its up to you, both the org name and the actual project name are up in the air.  `react-rad` might be neat.

@cristhiandas the biggest non-coding tasks are:

- organization management: someone has to add new interested parties, evaluate PRs and issues, etc.
- documentation: both soft-text and small videos
- workflow: what does a release cycle look like?  who does the builds?  who is "responsible" for making sure that people know to do the release?
- issue triage: are any of the outstanding issues and PRs relevant?  Use the existing items to build an action plan.

Working on an open source project is no different from a non-open source project, except for not knowing any of the contributors in real life.

**Comment by cristhiandas** [comment]
@spelunk cool! I'll start researching the best way to do all those stuff. I'll love to help on the documentation :)

**Comment by SheetJSDev** [comment]
https://github.com/reactide-community/reactide here's our fork, I invited interested people from this thread.

**Comment by spelunk** [comment]
https://github.com/reactide-community/reactide

**Comment by simonhochrein** [comment]
Should we create a slack channel?

**Comment by spelunk** [comment]
@simonhochrein https://github.com/reactide-community/reactide/issues/1 organization discussion here

**Comment by EP36** [comment]
We're currently looking for 3.0 beta testers!

**Comment by salmon258** [comment]
@EP36 is this project restarted?

**Comment by EP36** [comment]
@salmon258 why yes it is! We’re launching 3.0 today!

**Comment by simonhochrein** [comment]
It's so cool to see the project restarted. Good job!!

**Comment by salmon258** [comment]
@EP36 cool! I can be beta testers!

**Comment by EP36** [comment]
@salmon258 feel free to fork and try it out :)
